### PR TITLE
Service State

### PR DESCRIPTION
**Pull request by setrofim**
Add `GET /status` handling for `provisioning` and `verification`.
 
This addresses https://github.com/veraison/services/issues/46 and partially addresses https://github.com/veraison/services/issues/51

### CODE REVIEW

File: provisioning/api/handler.go [hunk 1]
```diff
@@ -50,6 +54,49 @@ const (
...
+	apiMediaTypeList, err := structpb.NewList([]interface{}{ProvisioningSessionMediaType})
+	if err != nil {
+		panic(err) // Should never happen as the value above is hard-coded.
+	}
+
+	decoderMediaTypeList, err := proto.NewStringList(o.DecoderManager.GetSupportedMediaTypes())
+	if err != nil {
+		ReportProblem(c,
+			http.StatusInternalServerError,
+			fmt.Sprintf("could not retrieve decoddr media types: %v", err),
```
Comment (thomas-fossati) [inline provisioning/api/handler.go:80]:
```suggestion
			fmt.Sprintf("could not retrieve decoder media types: %v", err),
```

**Review by thomas-fossati — APPROVED** [review]
LGTM!

File: provisioning/api/handler.go [hunk 2]
```diff
@@ -50,6 +54,49 @@ const (
 	ProvisioningSessionMediaType = "application/vnd.veraison.provisioning-session+json"
 )
 
+func (o *Handler) GetServiceState(c *gin.Context) {
+	state := proto.ServiceState{
+		ServerVersion: "0.0.1",
```
Comment (yogeshbdeshpande) [inline provisioning/api/handler.go:59]: Should the Server Version be hard coded or should it be set from a Config file? Just a question:

File: provisioning/api/handler.go (hunk 1 above)
Comment (yogeshbdeshpande) [inline provisioning/api/handler.go:80]:
```suggestion
			fmt.Sprintf("could not retrieve decoder media types: %v", err),
```

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM! Left few questions but looks ok

File: provisioning/api/handler.go (hunk 2 above)
Comment (thomas-fossati) [reply provisioning/api/handler.go:59]: I think version should be computed at build time from git metadata (tags, raw commit info) and assigned to a predefined global via `go build -w -X VERSION_VAR=VERSION_VAL`

Comment (thomas-fossati) [reply provisioning/api/handler.go:59]: Raised #51 to track this.

Comment (setrofim) [reply provisioning/api/handler.go]: No need :). Just pushed an update that sets versions from git.

Comment (yogeshbdeshpande) [reply provisioning/api/handler.go]: @setrofim Too cool 👍

**Review by thomas-fossati — APPROVED** [review]
awesome.

(In the commit msg we should note "Fix #46 and partially addresses #51")

**Pull request merged**